src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs

```diff
@@ -118,6 +118,7 @@ internal IEnumerable<IKeyValueAdapter> Adapters
         /// <summary>
         /// Specify what key-values to include in the configuration provider.
         /// <see cref="Select"/> can be called multiple times to include multiple sets of key-values.
+        /// If neither <see cref="Select"/> nor <see cref="SelectSnapshot"/> is ever called, all key-values with null label are included in the configuration provider.
```

Comment:
The intent seems good, but it's a bit odd that it's on the Select method. I guess I would expect it on AppConfigurationOptions summary.

Reply:
Yea I agree honestly, I wasn't sure if adding it to the options summary seemed random/too much information there. Probably a bit better than this